test/strings/basic.jl

```diff
@@ -112,11 +112,11 @@ end
 @test ""[10:9] == ""
 @test "hello"[10:9] == ""
 @test "hellø"[10:9] == ""
-@test SubString("hello", 1, 6)[10:9] == ""
+@test SubString("hello", 1, 5)[10:9] == ""
 @test SubString("hello", 1, 0)[10:9] == ""
-@test SubString("hellø", 1, 6)[10:9] == ""
+@test SubString("hellø", 1, 5)[10:9] == ""
 @test SubString("hellø", 1, 0)[10:9] == ""
-@test SubString("", 1, 6)[10:9] == ""
+# @test SubString("", 1, 6)[10:9] == ""
```

Comment:
Maybe just change this to `SubString("", 1, 1)`?

Reply:
`SubString("", 1, 1)` and `SubString("", 1, 6)` throw `BoundsError` as they are given invalid indices (like `getindex`).